docker/pulsar/pom.xml

```diff
@@ -79,7 +80,7 @@
                   <executable>${project.basedir}/../../pulsar-client-cpp/docker/build-wheels.sh</executable>
                   <arguments>
                     <!-- build python 3.8 -->
-                    <argument>3.8 cp38-cp38 manylinux2014 x86_64</argument>
+                    <argument>3.8 cp38-cp38 manylinux2014 ${pythonClientBuildArch}</argument>
```

Comment:
`os.detected.arch` may help on this topic. However, we name `aarch_64` returned by `os.detected.arch` as `aarch64` in `pulsar-client-cpp/docker/python-versions.sh` so there is a string mismatch.

cc @merlimat.

Reply:
It seems the root cause for this name mismatch comes from https://github.com/pypa/manylinux.

Reply:
For now this is fine. After the Python client is removed from the main repo, we can just do `pip install pulsar-client==2.10.1` and it will automatically select the right architecture.

Reply:
Yep. I'm OK to keep it as is now. From my perspective, the image is mainly used as a server, perhaps we can remove these clients from bundling later.

Reply:
@merlimat Thanks for your explanation. I noticed the comments just now:

```dockerfile
### Create 2nd stage from Ubuntu image
### and add OpenJDK and Python dependencies (for Pulsar functions)
```

Reply:
> it will automatically select the right architecture

It's cool as it can significantly reduce the image build time and avoid qemu to compile C++ library if we want to integrate it with GitHub Actions.